themes/bootstrap5/templates/admin/menu.phtml

```diff
@@ -1,11 +1,5 @@
 <nav class="list-group">
-  <a href="<?=$this->url('admin')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'home' ? ' active' : ''?>"><?=$this->transEsc('Home')?></a>
-  <a href="<?=$this->url('admin/social')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'socialstats' ? ' active' : ''?>"><?=$this->transEsc('Social Statistics')?></a>
-  <a href="<?=$this->url('admin/config')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'config' ? ' active' : ''?>"><?=$this->transEsc('Configuration')?></a>
-  <a href="<?=$this->url('admin/maintenance')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'maintenance' ? ' active' : ''?>"><?=$this->transEsc('System Maintenance')?></a>
-  <a href="<?=$this->url('admin/tags')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'tags' ? ' active' : ''?>"><?=$this->transEsc('Tag Maintenance')?></a>
-  <a href="<?=$this->url('admin/feedback')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'feedback' ? ' active' : ''?>"><?=$this->transEsc('Feedback Management')?></a>
-  <?php if ($this->overdrive()->showOverdriveAdminLink()):?>
-    <a href="<?=$this->url('admin/overdrive')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'overdrive' ? ' active' : ''?>"><?=$this->transEsc('od_admin_menu')?></a>
-  <?php endif; ?>
+  <?php foreach ($this->items as $item): ?>
```

Comment:
Is it worth implementing menu groups here parallel to the myresearch menu? Would doing that allow you to share more code between the menu-related view helpers (i.e. reduce differentiated render methods)?

Reply:
My thinking was that this is the model part of the refactoring and the view part would be a separate PR, especially as I am not sure how big a change that would be. But if you would prefer for everything to be done in one PR, I can give it a shot!

Reply:
That makes sense; as I said above, I'm happy to keep this simple as long as we follow up with consistency as a next step.